l10n/en/firefox/sync.ftl

```diff
@@ -8,6 +8,8 @@ firefox-sync-firefox-sync = { -brand-name-firefox-sync }
 
 firefox-sync-access-and-sync = Access and sync your bookmarks, passwords, open tabs and more — everywhere you use { -brand-name-firefox }.
 
+firefox-sync-firefox = { -brand-name-firefox } browser
```

Comment:
There's a `-brand-name-firefox-browser` you could use here, or it could just be "Firefox"